### PR TITLE
Fix doctests in utils.py

### PR DESCRIPTION
**Pull request by jacobtomlinson**
I was about to start some TDD with `pytest --doctest-modules distributed/utils.py` and noticed there were some failures.

### CODE REVIEW

File: distributed/utils.py [hunk 1]
```diff
@@ -1451,7 +1453,7 @@ def import_term(name: str):
 
     Examples
     --------
-    >>> import_term("math.sin")
+    >>> import_term("math.sin") # doctest: +SKIP
```
Comment (jacobtomlinson) [inline distributed/utils.py:1456]: I chose to skip here because the actual result was `<built-in function sin>`. But we could always go the other way.

**Review by fjetter — APPROVED** [review]
Big +1 for TDD

File: distributed/utils.py (hunk 1 above)
Comment (fjetter) [reply distributed/utils.py:1456]: I'm fine either way. I personally would probably only skip if it cannot be executed for some reason but I don't think this is really crucial

**Comment by jacobtomlinson** [comment]
Failing test here appears to be unrelated. I'm going to go ahead and merge as it's only a docstring fix.

**Pull request merged**